backend/src/main/java/com/bakdata/conquery/models/forms/managed/RelativeFormQueryPlan.java

```diff
@@ -66,22 +67,37 @@ public EntityResult execute(QueryExecutionContext ctx, Entity entity) {
 		// We look at the first result line to determine the length of the subresult
 		int featureLength = featureResult.getValues().get(0).length;
 		int outcomeLength = outcomeResult.getValues().get(0).length;
-		// Whole result is the concatenation of the subresults. However the sub result includes the date restriction which we drop.
-		int size = featureLength + outcomeLength - 2;
-
-		// merge the full (index == null) lines
-		Object[] mergedFull = new Object[size];
-		setFeatureValues(mergedFull, featureResult.getValues().get(0));
-		setOutcomeValues(mergedFull, outcomeResult.getValues().get(0), featureLength);
-		values.add(mergedFull);
+		
+		/*
+		 *  Whole result is the concatenation of the subresults. The final output format combines resolution info, index and eventdate of both sub queries.
+		 *  The feature/outcome sub queries are of in form of: [RESOLUTION], [INDEX], [EVENTDATE], [FEATURE/OUTCOME_DR], [FEATURE/OUTCOME_SELECTS]... 
```

Comment:
It's good to explain this. Would even be better if things like these would be documented in the code itself. So could this e.g. a special type / class / enum?